RecoMET/METProducers/python/CaloRecHitsBeamHaloCleaned_cfi.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+CaloRecHitsBeamHaloCleaned = cms.EDProducer('CaloRecHitsBeamHaloCleaned',
```

Comment:
actually, it's better not to make this file at all and use what the system will generate based on the fillDescriptions defaults: it will make caloRecHitsBeamHaloCleaned_cfi.py file in $CMSSW_BASE/python